test/authz/testing.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // Policy is a test rego policy for a token based authz system
-const Policy = `package restauthz
+const Policy = `package policy.restauthz
```

Comment:
Just curious why this change was needed.

Reply:
If I remember correctly it was because we needed to push data into the store as part of the tests, like each test has a set of "external data" plus the common policy. To allow the tests to do a PUT to /v1/data/whatever and use bundles I moved the policies in the bundle into their own sub-path and defined roots for the bundle specific to them.